File: dart/tools/latn_iast_transliteration_verification/gujr_to_latn_iast_test.dart
// ignore_for_file: avoid_print

import 'package:indic_script_converter/brahmic_to_latn_iast.dart';
import 'example_gujr.dart' as examples;

void main() {
  print('----------------------------------------------------------------');
  print(' GUJARATI TO LATN IAST TRANSLITERATION');
  print('----------------------------------------------------------------');

  for (final source in examples.gujaratiSmokeSamples) {
    final result = source.toCanonicalIastFromGujarati();

    print('"$source" -> "$result"');
  }
}

File: python/tools/latn_iast_transliteration_verification/gujr_to_latn_iast_test.py
from __future__ import annotations

from lipimala.brahmic_to_latn_iast import to_canonical_iast_from_gujarati
from tools.fixtures.example_gujr import gujarati_smoke_samples


def main() -> None:
    print('-' * 64)
    print(' GUJARATI TO LATN IAST TRANSLITERATION')
    print('-' * 64)
    for source in gujarati_smoke_samples:
        result = to_canonical_iast_from_gujarati(source)
        print(f'"{source}" -> "{result}"')


if __name__ == '__main__':
    main()
